### DISCUSSION TITLE
Configuration option to assign the user who forced the creation of a PR as a reviewer

### DISCUSSION THREAD

**Discussion by Egorand** [renovatebot/renovate#22553] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

TLDR: I'm looking for a way to configure Renovate such that if a Github user ticks the box on the dependency dashboard to force-create a PR for a specific dependency, Renovate adds that Github user to the reviewers list.

We've got a set of internal dependencies that get released very frequently, about once every 30 minutes. We rate-limit the automatic creation of PRs for those dependencies to "once a day", so for the most time these dependencies appear in the "Rate-Limited" section of the dependency dashboard. Whenever an engineer on our team needs to pull in the latest version of a dependency, they're supposed to force-create the upgrade PR by ticking the box in the dashboard. Our default set of reviewers includes the auto-approver bot and the team that maintains the Renovate integration, so the engineer who created the PR would not get notified by email when the PR is opened. We're looking for a way to have that person notified of that specific PR, without adding them to the default reviewers list.

### What you tried so far.

I searched through the ["Configuration Options"](https://docs.renovatebot.com/configuration-options/), but couldn't find any that would fit our use case.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
This is an interesting idea, although would be challenging to implement.

If you consider the regular Renovate CLI, it would mean that Renovate needs a way to query the edit history of the issue, parse that, and then map it to what was ticked or not - keeping in mind that multiple people could tick checkboxes.

**Reply by Egorand** [reply]
Makes sense, thanks! Are there any existing configuration options you could suggest to address our problem?

**Reply by rarkins** [reply]
The closest might be to have a codeowners file so that people can be assigned from codeowners

**Comment by oldergod** [comment]
I ran into this today. Let's address it.
